### ISSUE TITLE
Cannot look up Chinese characters

### ISSUE THREAD

**Issue by null-von-sushi** [johnfactotum/quick-lookup#32]
It seems that contrary to the description, it is not currently possible to look up another language's words.
For example, [the English Wiktionary's entry for 仁](https://en.wiktionary.org/wiki/%E4%BB%81).
The app just says that nothing was found.

**Comment by johnfactotum** [comment]
Yes, it's a known issue of the upstream Wiktionary API. See https://github.com/johnfactotum/quick-lookup/issues/13#issuecomment-739115045. Note that the bug only affects single character words. It works for words.
